content/en/device-support/_index.md

```diff
@@ -219,6 +219,29 @@ Passkeys created in **macOS** can be used on:
                                 alt="calendar icon"></i><br>Planned
                         </td>
                     </tr>
+                    <tr class="align-middle">
+                        <td class="fw-bold">Support in Native Apps</td>
+                        <td class="text-center"><i class="bi bi-wrench-adjustable-circle-fill" title="Beta"
```

Comment:
This should be just green check, Android 9+. The new CredMan is in Android 14, but you can still use the old APIs in prior versions. 

I can add caveats to the Android reference page if needed.